console/encrypter.py

```diff
@@ -0,0 +1,88 @@
+from cryptography.hazmat.backends.openssl.backend import backend
+from cryptography.hazmat.primitives import serialization, hashes
+from cryptography.hazmat.primitives.asymmetric import padding
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+from cryptography.hazmat.primitives.serialization import Encoding, PublicFormat
+
+import base64
+import os
+from console import settings
+import jwt
+
+KID = 'SDE'
+
+
+class Encrypter (object):
+    def __init__(self):
+        private_key_bytes = self._to_bytes(settings.EQ_PRIVATE_KEY)
```

Comment:
Can we avoid relying on settings files inside class initialisation? This isn't a great way to approach our microservices architecture. Instead, load the thing into the app's env and use that - more configurable and cloudfoundry compatible.